oxd-server/src/main/java/org/xdi/oxd/server/RestResource.java

```diff
@@ -196,7 +226,8 @@ public String getRp(@HeaderParam("Authorization") String authorization, String p
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
     public String getJwks(@HeaderParam("Authorization") String authorization, String params) {
-        return process(CommandType.GET_JWKS, params, GetJwksParams.class, authorization);
+        final ObjectMapper mapper = new ObjectMapper().configure(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_AS_NULL,true);
+        return processWithCustomMapper(CommandType.GET_JWKS, params, GetJwksParams.class, authorization, mapper);
```

Comment:
Now we have 2 methods for processing commands, lets keep single method and pass mapper inside the `process`, 
```
private static <T extends IParams> String process(ObjectMapper mapper, CommandType commandType, String paramsAsString, Class<T> paramsClass, String authorization) {
    ...
}
```

Also overload `process` method with mapper that suites for 90% of commands, like 
```
private static <T extends IParams> String process(ObjectMapper mapper, CommandType commandType, String paramsAsString, Class<T> paramsClass, String authorization) {
    return process(CoreUtils.createJsonMapper(), commandType, paramsAsString, paramsClass, authorization);
}
```

Reply:
The two mappers are from different package. If we migrate CoreUtils mapper to new one,  it break some other code.

Reply:
we must stick to one single mapper from oxauth-client. We don't want to clash with it.